core/src/main/scala/org/broadinstitute/dsde/rawls/user/UserService.scala

```diff
@@ -441,30 +503,32 @@ class UserService(protected val userInfo: UserInfo, val dataSource: SlickDataSou
     }
   }
 
-  def startBillingProjectCreation(createProjectRequest: CreateRawlsBillingProjectFullRequest): Future[PerRequestMessage] = {
-    for {
-      _ <- validateCreateProjectRequest(createProjectRequest)
-      _ <- checkServicePerimeterAccess(createProjectRequest.servicePerimeter)
-      billingAccount <- checkBillingAccountAccess(createProjectRequest.billingAccount)
-      result <- internalStartBillingProjectCreation(createProjectRequest, billingAccount)
-    } yield result
-  }
+  def updateBillingProjectBillingAccount(billingProjectName: RawlsBillingProjectName, updateAccountRequest: UpdateRawlsBillingAccountRequest): Future[PerRequestMessage] = {
+    validateBillingAccountName(updateAccountRequest.billingAccount.value)
 
-  def updateBillingAccount(projectName: RawlsBillingProjectName, updateAccountRequest: UpdateRawlsBillingAccountRequest): Future[PerRequestMessage] = {
-    requireProjectAction(projectName, SamBillingProjectActions.updateBillingAccount) {
+    requireProjectAction(billingProjectName, SamBillingProjectActions.updateBillingAccount) {
       for {
         hasAccess <- gcsDAO.testBillingAccountAccess(updateAccountRequest.billingAccount, userInfo)
         _ = if (!hasAccess) {
           throw new RawlsExceptionWithErrorReport(ErrorReport(StatusCodes.BadRequest, "Billing account does not exist, user does not have access, or Terra does not have access"))
         }
-        result <- updateBillingAccountInternal(projectName, Option(updateAccountRequest.billingAccount))
+        result <- updateBillingAccountInternal(billingProjectName, Option(updateAccountRequest.billingAccount))
```

Comment:
In `develop` we make the actual call to Google to update the billing account when `updateBillingProjectBillingAccount` is called. In a PPW world, we allow the `WorkspaceBillingAccountMonitor` to handle those Google calls for us. All that we need to do is encompassed in `updateBillingAccountInternal` which does not make any Google calls.